runtime/exporter-otlp/src/test/java/io/aklivity/zilla/runtime/exporter/otlp/internal/config/OtlpExporterConfigTest.java

```diff
@@ -138,4 +138,70 @@ public void shouldOverrideRelativeMetricsUrl()
         assertThat(metrics, equalTo(URI.create("http://example.com/v42/metrix")));
         assertThat(logs, equalTo(URI.create("http://example.com/v42/logz")));
     }
+
```

Comment:
Please add a test for full URI override, including scheme host and port that differ from base location.